stubs/stripe/stripe/api_resources/abstract/createable_api_resource.pyi

```diff
@@ -6,10 +7,10 @@ from stripe.api_resources.abstract.api_resource import APIResource as APIResourc
 class CreateableAPIResource(APIResource):
     @classmethod
     def create(
-        cls,
+        cls: type[Self],
         api_key: Any | None = ...,
         idempotency_key: Any | None = ...,
         stripe_version: Any | None = ...,
         stripe_account: Any | None = ...,
         **params,
-    ): ...
+    ) -> Self: ...
```

Comment:
Any tips on how to review this change? The code for this method is a bit of a maze, and playing around in the REPL seems sort-of nontrivial as well, for this library.